tools/batch_run_pipeline_qc_face_script.pl

```diff
@@ -0,0 +1,193 @@
+
+=head1 SYNOPSIS
+
+perl batch_run_pipeline_qc_face_script.pl [-profile file] [-out_basedir directory] < list_of_files.txt
+
+Available options are:
+
+-profile: name of config file in ../dicom-archive/.loris_mri (typically called prod)
+
+-out_basedir: path to the output base directory where the jpg will be created
+
+=head1 DESCRIPTION
+
+This script creates the 3D rendering QC images on multiple MINC files.
+The list of MINC files to use to generate those 3D JPEG images are provided
+through a text file (e.g. C<list_of_files.txt> with one file path per line.
+
+An example of what a C<list_of_files.txt> might contain for 3 files to use to
+create a 3D JPEG rendering of a scan:
+to be defaced:
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_001_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_002_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t2w_001_t2w-defaced_001.mnc
+
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative Neuroscience
+
+=cut
+
+
+use strict;
+use warnings;
+no warnings 'once';
+use Getopt::Tabular;
+use File::Basename;
+use NeuroDB::DBI;
+use NeuroDB::ExitCodes;
+
+
+
+
+#############################################################
+## Create the GetOpt table
+#############################################################
+
+my $profile;
+my $out_basedir;
+
+my @opt_table = (
+    [ '-profile',     'string', 1, \$profile,     'name of config file in ../dicom-archive/.loris_mri'              ],
+    [ '-out_basedir', 'string', 1, \$out_basedir, 'path to the output base directory where the jpg will be created' ] 
+);
+
+my $Help = <<HELP;
+*******************************************************************************
+Run pipeline_qc_face.pl in batch mode
+*******************************************************************************
+
+This script creates the 3D rendering QC images on multiple MINC files. 
+The list of MINC files to use to generate those 3D JPEG images are provided 
+through a text file (e.g. C<list_of_files.txt> with one file path per line.
+
+An example of what a C<list_of_files.txt> might contain for 3 files to use to 
+create a 3D JPEG rendering of a scan:
+to be defaced:
+
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_001_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_002_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t2w_001_t2w-defaced_001.mnc
+
+Documentation: perldoc batch_run_pipeline_qc_face_script.pl
+
+HELP
+
+my $Usage = <<USAGE;
+usage: ./batch_run_pipeline_qc_deface_script.pl -profile prod  < list_of_files.txt [options]
```

Comment:
Missing `-outbasedir` and I would remove `[options]`

Reply:
done